addon/PageWorker.js

```diff
@@ -27,7 +28,8 @@ class PageWorker {
     });
     // Note: According to the redux docs, calling .subscribe on a store
     // returns a function which will unsubscribe
-    this._unsubscribe = this._store.subscribe(this._onDispatch);
+    const w = watch(this._store.getState);
+    this._unsubscribe = this._store.subscribe(w(this._onDispatch));
```

Comment:
I'm confused.  Since _onDispatch doesn't use any of the arguments that redux-watch has w pass, what is the point of adding redux-watch?